src/main/java/com/mraof/minestuck/tileentity/redstone/SummonerTileEntity.java

```diff
@@ -108,12 +109,23 @@ public EntityType<?> getSummonedEntity()
 		return this.summonType;
 	}
 	
+	public void setSummonRange(int rangeIn)
+	{
+		this.summonRange = rangeIn;
```

Comment:
Consider if you want to enforce a range here with `MathHelper.clamp()` for safety.